proto/minder/v1/minder.proto

```diff
@@ -2626,3 +2626,42 @@ message EvaluationHistoryAlert {
     // the structure and contents are alert specific, and are subject to change.
     string details = 2;
 }
+
+message SelectorEntity {
+    // one of repository, pull_request, artifact (see oneof entity)
+    Entity entity_type = 1;
+    // the name of the entity, same as the name in the entity message
+    string name = 2;
+
+    oneof entity {
+        SelectorRepository repository = 5;
+        SelectorArtifact artifact = 6;
+    }
+
+    message SelectorRepository {
+        // the full name of the repository, e.g. stacklok/minder
+        string name = 1;
+        // the name of the repository, e.g. minder
+        string repo_name = 2;
+        // the owner of the repository, e.g. stacklok
+        string owner = 3;
+
```

Comment:
Only the full name is used now, formatted by the provider. I dropped the repo_name and owner as separate fields. I named the attribute just `name` because I just think that's more user-friendly but I'm not attached to `name` vs. `full_name`.